### PR TITLE
Add @IAmDrMonkey Bio

### PR DESCRIPTION
**Pull request by iamdrmonkey**
This pull request adds the shiny, metal bio for @iamdrmonkey related to #3.

### CODE REVIEW

**Comment by mbarbre1** [comment]
What language does a code monkey use?

File: docs/iamdrmonkey-bio.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Behold! I AM Dr. Monkey!
+
+* **Name:** Mike Dormany
+* **Occupation:** Senior Code Monkey
+* **Hometown:** Portland, OR (I ```put``` birds on things)
```
Comment (nathos) [inline docs/iamdrmonkey-bio.md]: Do you also pickle things?

**Comment by iamdrmonkey** [comment]
@mbarbre1 I use "Monkey C" :)

File: docs/iamdrmonkey-bio.md (hunk 1 above)
Comment (iamdrmonkey) [inline docs/iamdrmonkey-bio.md]: I'm constantly pickled. It makes my code more interesting.

**Pull request merged**